CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.23.0 Release
+- Release Date: September 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Bug Fixes
+ - Fixed an issue where some Giphy actions were not working in ephemeral posts on mobile.
+ - Fixed an issue where the download step was missing when previewing a video right after posting it. 
```

Comment:
There is no download step when the video was posted from the mobile app. It was intentionally removed as you don’t need to download something that you just uploaded